drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -2325,6 +2325,8 @@ void RasterizerStorageGLES3::_update_shader(Shader *p_shader) const {
 			shaders.actions_canvas.render_mode_values["blend_mul"] = Pair<int *, int>(&p_shader->canvas_item.blend_mode, Shader::CanvasItem::BLEND_MODE_MUL);
 			shaders.actions_canvas.render_mode_values["blend_premul_alpha"] = Pair<int *, int>(&p_shader->canvas_item.blend_mode, Shader::CanvasItem::BLEND_MODE_PMALPHA);
 			shaders.actions_canvas.render_mode_values["blend_disabled"] = Pair<int *, int>(&p_shader->canvas_item.blend_mode, Shader::CanvasItem::BLEND_MODE_DISABLED);
+			shaders.actions_canvas.render_mode_values["blend_min"] = Pair<int *, int>(&p_shader->canvas_item.blend_mode, Shader::CanvasItem::BLEND_MODE_MIN);
+			shaders.actions_canvas.render_mode_values["blend_max"] = Pair<int *, int>(&p_shader->canvas_item.blend_mode, Shader::CanvasItem::BLEND_MODE_MAX);
```

Comment:
50 lines below this change, the equivalent change must be made to shaders.actions_scene to add the same constants for 3d shaders.

Reply:
I suspect this would also require changes to e.g. `RasterizerSceneGLES3::_render_list`, as the setup code is only done for canvas shaders. Although I don't know whether the intention was to make this available in 3d too.

Reply:
Yes it should be available in 3D too! I just have not gotten around fully to finish it :D